versioned_docs/version-2.0-2.4/pages-for-subheaders/backup-restore-and-disaster-recovery.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Backups and Disaster Recovery
+keywords: [rancher v2.0-v2.4 backup restore, rancher v2.0-v2.4 backup and restore, backup restore rancher v2.0-v2.4, backup and restore rancher v2.0-v2.4]
```

Comment:
Users are probably more likely to search for their version number rather than a range.

Reply:
Including the same terms with each individual version number four times over would get very verbose, though. I think using the range is fine for this PR.